force-app/main/default/classes/ProgramSettingsController.cls

```diff
@@ -146,6 +146,22 @@ public with sharing class ProgramSettingsController {
         }
     }
 
+    /**
+     * @description Deletes auto enrollment mapping
+     * @param mappingName The name of the original auto enrollment mapping
+     * @return The account record type of the mapping deleted or null if no mapping found.
+     */
+    @AuraEnabled
+    public static String deleteAutoEnrollmentMappings(String mappingName) {
+        try {
+            AutoEnrollmentMappingModel deletedMappingModel = ProgramSettingsController.locateAutoEnrollmentMappingService()
+                .deleteAutoEnrollmentMappings(mappingName);
+            return deletedMappingModel == null ? null : deletedMappingModel.accountRecordTypeFieldValue;
```

Comment:
I am a little not sure on this. If we can not find a mapping, we shouldn't tell users that the account record type mapping is deleted right? fyi @MatthewBlanski

Reply:
I asked about this before, it seems like UI error handling/notification is another user story. I made this code similar to Affilication Mapping delete which also returns null if a record was not found. I think this should be handled in the UI